helix-view/src/editor/variable_expansion.rs

```diff
@@ -0,0 +1,159 @@
+use crate::Editor;
```

Comment:
btw I didn't know why we are not using regex here. I think it could make the code way much clearer, but I kept it the way it was done in the original PR (my comment is about the whole file)